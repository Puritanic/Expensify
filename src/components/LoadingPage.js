import React from 'react';

const LoadingPage = () => (
  <div className="loader">
    <img className="loader__image" src="/images/loader.gif" alt="App loading" />
  </div>
);

export default LoadingPage;
