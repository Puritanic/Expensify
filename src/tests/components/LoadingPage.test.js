import React from 'react';
import LoadingPage from '../../components/LoadingPage';

/* eslint-disable no-undef */
it('should render LoadingPage without crashing', () => {
  const wrapper = shallow(<LoadingPage />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
