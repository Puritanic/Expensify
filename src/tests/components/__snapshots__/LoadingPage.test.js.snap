// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoadingPage without crashing 1`] = `
<div
  className="loader"
>
  <img
    alt="App loading"
    className="loader__image"
    src="/images/loader.gif"
  />
</div>
`;
